anchorecli/cli/utils.py

```diff
@@ -299,10 +299,11 @@ def format_output(config, op, params, payload):
                 obuf = obuf + "\n"
             else:
                 if params['query_type'] == 'os':
-                    header = ['Package', 'Version', 'License']
```

Comment:
watch out though... even if anchore-cli is released in the same container as anchore-engine, it is possible to install anchore-cli separately and run it against an older version of anchore-engine. This change would break that compatibility. 

Could you support both? And add a comment to deprecate it in the next release, and maybe even a `DeprecationWarning` as well.

Reply:
The line commented on is only for the header; I didn't see a way to specify `--json` (or similar) where the output could be ingested by automation (and could be dependent on a "license" field). 

[This line ensures backwards compatability](https://github.com/anchore/anchore-cli/blob/ca5fb2ee2dc0d5be52911f6e9c9ae323c5a475ce/anchorecli/cli/utils.py#L305), preferring the new `licenses` field over the old `license` field, but still gracefully degrading.